src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -132,25 +130,55 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             },
             handleError: "{that}.events.onError.fire",
             handlePause: {
-                changePath: "paused",
-                value: true
+                funcName: "fluid.textToSpeech.handlePause",
+                args: ["{that}"]
             },
             handleResume: {
-                changePath: "paused",
-                value: false
+                funcName: "fluid.textToSpeech.handleResume",
+                args: ["{that}"]
             }
         }
     });
 
+    // Issue commands to the speechSynthesis interface after a delay; this
+    // makes the wrapper behave somewhat better when issuing commands, especially
+    // play and pause
+    fluid.textToSpeech.asyncSpeechSynthesisControl = function (control, delay, args) {
+        setTimeout(function () {
+            speechSynthesis[control](args);
+        }, delay);
+    };
+
     fluid.textToSpeech.speak = function (that, speaking) {
         that.events[speaking ? "onStart" : "onStop"].fire();
     };
 
-    fluid.textToSpeech.pause = function (that, paused) {
-        if (paused) {
-            that.events.onPause.fire();
-        } else if (that.model.speaking) {
-            that.events.onResume.fire();
+    fluid.textToSpeech.handlePause = function (that) {
+        that.applier.change("paused", true);
+        that.events.onPause.fire();
+        // Clear to issue any waiting resume command
+        fluid.textToSpeech.clearControlRequest(that, "resumeRequested", "resume");
+    };
+
+    fluid.textToSpeech.handleResume = function (that) {
+        that.applier.change("paused", false);
+        that.events.onResume.fire();
+        // Clear to issue any waiting pause command
+        fluid.textToSpeech.clearControlRequest(that, "pauseRequested", "pause");
+    };
+
+    fluid.textToSpeech.clearControlRequest = function (that, modelBoolPath, controlName) {
+        if(fluid.get(that.model, modelBoolPath)) {
```

Comment:
I've got my pre-commit linter running now on my Infusion project, so it runs (and aborts the commit on any linting errors) before every commit. Whatever the issue the `.eslintrc.json` config in Infusion isn't catching it. (made sure to update to latest upstream master before saying this...)